application/core/LS_Twig_Extension.php

```diff
@@ -396,4 +396,16 @@ public static function listScriptFiles()
         }
     }
 
+    /**
+     * Process any string with current page
+     * @param string to be processed
+     * @param boolean $static return static string (or not)
+     * @param integer $numRecursionLevels recursion (max) level to do
+     * @param array $aReplacement replacement out of EM
+     * @return string
+     */
+    public static function processString($string,$static=false,$numRecursionLevels=3,$aReplacement = array())
```

Comment:
Thing static is more used than recursion, and aReplacemnt than recursion . Then theme dev have less optioon to write